### PR TITLE
Riva deployment guides with gke & aks

### PR DESCRIPTION
**Pull request by sbagdiya28**

### CODE REVIEW

**Review by anandagrawal2909 — COMMENTED** [review]
>>line#154 
You may want to avoid mentioning AWS here ?  "AWS Classic Load Balancer"

**Review by anandagrawal2909 — COMMENTED** [review]
Rest looks good

**Review by anandagrawal2909 — APPROVED** [review]

**Review by anandagrawal2909 — APPROVED** [review]
Changes were validated as part of bug https://nvbugs/4113308 [Riva-Skills] Need to add and update steps in doc for riva deployment using GKE and AKS.

**Comment by rmittal-github** [comment]
@LynseyFabel for review

File: deploy-gke.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    helm fetch traefik/traefik
+    tar -zxvf traefik-*.tgz
+    ```
+
+2.  Modify the `traefik/values.yaml` file.
+
+    1. Change `service.type` from `LoadBalancer` to `ClusterIP`. This exposes the service on a cluster-internal IP.
+
+    2. Set `nodeSelector` to `{  cloud.google.com/gke-nodepool: cpu-linux-lb }`. Similar to what you did for the Riva API service,
+    this tells the Traefik service to run on the `cpu-linux-lb` nodepool.
```
Comment (LynseyFabel) [inline deploy-gke.md:167]:
```suggestion
    this tells the Traefik service to run on the `cpu-linux-lb` node pool.
```

File: deploy-aks.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    helm fetch traefik/traefik
+    tar -zxvf traefik-*.tgz
+    ```
+
+2.  Modify the `traefik/values.yaml` file.
+
+    1. Change `service.type` from `LoadBalancer` to `ClusterIP`. This exposes the service on a cluster-internal IP.
+
+    2. Set `nodeSelector` to `{  kubernetes.azure.com/agentpool: loadbalancer}`. Similar to what you did for the Riva API service,
+    this tells the Traefik service to run on the `loadbalancer` nodepool.
```
Comment (LynseyFabel) [inline deploy-aks.md:183]:
```suggestion
    this tells the Traefik service to run on the `loadbalancer` node pool.
```

File: deploy-aks.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+4.  From inside the shell of the client pod, run the sample ASR client on an example `.wav` file. Specify the `traefik.default.svc.cluster.local` endpoint, with port 80, as the service address.
+    ```bash
+    riva_streaming_asr_client \
+       --audio_file=wav/en-US_sample.wav \
+       --automatic_punctuation=true \
+       --riva_uri=traefik.default.svc.cluster.local:80
+    ```
+
+## Scaling the cluster
```
Comment (LynseyFabel) [inline deploy-aks.md:274]:
```suggestion
## Scaling the Cluster
```

**Review by LynseyFabel — COMMENTED** [review]
My review is complete. I've left a few comments.

File: deploy-gke.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+3. If you see that gpu driver plugin is not Enabled by GCP, Then deploy it using below command:
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+    ```
+
+4. Verify gpu plugin installation with any of the following command:
+    ```bash
+    kubectl get pod -A | grep nvidia
+
```
[the remark this answers is not in the document]
Comment (sbagdiya28) [reply deploy-gke.md:119]: yes will add.

**Comment by sbagdiya28** [comment]
@LynseyFabel updated the guides as per the comments from you

**Review by rmittal-github — APPROVED** [review]

**Pull request merged**